meshio/exodus_io.py

```diff
@@ -80,6 +80,9 @@ def read(filename):
     point_data_names = []
     pd = []
     cells = {}
+    ns_names = []
+    ns = []
```

Comment:
-> `node_sets`. Ah no wait, that name is already taken. What's the difference between the node set dictionary `{}` and this list here? Perhaps one of the two should get a different name.

Reply:
you means the temporary name 'ns' ? I think it is not important just like you use 'pd' above as point_data. I just follows your naming style. Please modify its name as you like.

Reply:
Hehe, good point. When reviewing PRs I tend to be more catholic than the pope.